rogden/tailwind-config-viewer
# Upgrade to vue 3

Vue 2 reached end of life in December, so this upgrades the package to Vue 3 following the [vue CLI migration instructions][1]. Moving to vue 3 gets us onto Vite. The priorities for this work were to keep the local npm scripts running while also making sure that the cli still worked when the package is used as a dependency.

I've also bumped the tailwind version.

To test these changes I did the following:

- ran `npm run dev` in the package and verified that the page looked fine in the browser with the sample config
- ran `./cli/index.js -c tailwind.config.sample.js` to simulate using the cli when the package is installed
- used the cli while `yarn linked` to another local package to run the server and view the config in a browser
- used the cli to `export` the viewer and served the static pages from the output directory with `npx http-serve .`

Dark mode still works, and I was able to see the config values that I expected to see.

[1]: https://v3-migration.vuejs.org/recommendations.html#build-toolchain
